src/main/scala/com/databricks/spark/redshift/RedshiftJDBCWrapper.scala

```diff
@@ -0,0 +1,210 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.databricks.spark.redshift
+
+import java.sql.{Connection, DriverManager, ResultSetMetaData, SQLException}
+import java.util.Properties
+
+import scala.util.Try
+
+import org.apache.spark.Logging
+import org.apache.spark.sql.types._
+
+/**
+ * Shim which exposes some JDBC helper functions.
+ */
+private[redshift] class JDBCWrapper extends Logging {
+
+  def registerDriver(driverClass: String): Unit = {
```

Comment:
Can you add a comment about what is going on here and why we are using reflection.